src/rgw/rgw_lc.cc

```diff
@@ -313,6 +313,7 @@ int RGWLC::bucket_lc_prepare(int index, LCWorker* worker)
 
 #define MAX_LC_LIST_ENTRIES 100
   do {
+    entries.clear();
```

Comment:
This should actually be in RadosLifecycle::list_entries(), I think, at the top.

Reply:
@dang I modified it as you commented, and pushed it again.